subsys/bluetooth/audio/tbs.c

```diff
@@ -1441,40 +1335,31 @@ static ssize_t write_call_cp(struct bt_conn *conn,
 		notify_ccp(conn, attr, call_index, ccp->opcode, status);
 	} /* else local operation; don't notify */
 
-	if (inst != NULL && status == BT_TBS_RESULT_CODE_SUCCESS) {
-		notify_calls(inst);
-		notify_app(conn, inst, len, ccp, status, call_index);
+	if (tbs != NULL && status == BT_TBS_RESULT_CODE_SUCCESS) {
+		notify_calls(tbs);
+		notify_app(conn, tbs, len, ccp, status, call_index);
 	}
 
 	return len;
 }
 
 static void call_cp_cfg_changed(const struct bt_gatt_attr *attr, uint16_t value)
 {
-	const struct tbs_service_inst *inst = lookup_inst_by_ccc(attr);
+	const struct service_inst *inst = lookup_inst_by_attr(attr);
 
 	if (inst != NULL) {
-		LOG_DBG("Index %u: value 0x%04x", inst->index, value);
-	} else if (IS_ENABLED(CONFIG_BT_GTBS)) {
-		LOG_DBG("GTBS: value 0x%04x", value);
+		LOG_DBG("Index %u: value 0x%04x", inst_index(inst), value);
 	}
 }
 
 static ssize_t read_optional_opcodes(struct bt_conn *conn,
 				     const struct bt_gatt_attr *attr,
 				     void *buf, uint16_t len, uint16_t offset)
 {
-	uint16_t optional_opcodes;
+	struct service_inst *inst = BT_AUDIO_CHRC_USER_DATA(attr);
```

Comment:
```suggestion
	const struct service_inst *inst = BT_AUDIO_CHRC_USER_DATA(attr);
```

I think most/all of the `read` can use `const`